src/Components/CreateImageWizard/CreateImageWizard.tsx

```diff
@@ -606,7 +606,6 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
             name="Details"
             id="step-details"
             navItem={customStatusNavItem}
-            status={detailsValidation.disabledNext ? 'error' : 'default'}
```

Comment:
If we remove this we'll lose the status indication of Details step, so this needs to stay.

Reply:
thanks for that, I know but we have kind of wired bug, 
when we open the create image screen for the first time, we see that this step appears with error, maybe it takes few second until we get the value in blueprint name field?
or maybe it is related to https://github.com/osbuild/image-builder-frontend/issues/2929?

Reply:
This is connected to fact that we populate the name of blueprint after getting to the step. Meaning there is an empty value for blueprint name in all the previous steps, thus the error icon in navigation.

I can think of two possible ways of how to resolve that - either we could update details step validation or `customStatusNavItem` in CreateImageWizard.tsx to reflect that there's a bit more unique pattern in place.

Or we could trigger the blueprint name generation earlier so the field gets filled in and the validation doesn't trip on the empty field.